### PR TITLE
BOTMETA: Create a $team_scaleway and add Spredzy in it

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

BOTMETA: Create a $team_scaleway and add Spredzy in it

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

  - N/A

##### ANSIBLE VERSION

  - N/A

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -950,8 +950,9 @@ files:
   $module_utils/jboss/:
     maintainers: $team_jboss
   $module_utils/scaleway.py: &scaleway
-    maintainers: sieben hekonsek
+    maintainers: $team_scaleway
     labels:
+      - scaleway
```
Comment (pilou-) [inline .github/BOTMETA.yml:955]: The `scaleway` GitHub label doesn't exist yet and needs to be manually created. @resmo could you create it please ?

Comment (resmo) [reply .github/BOTMETA.yml:955]: i would like to forward it to the repo owners. /cc @gundalow

Comment (gundalow) [reply .github/BOTMETA.yml:955]: Created https://github.com/ansible/ansible/labels/scaleway

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
May take the bot a few days to walk over existing issues & PRs

**Pull request merged**